packages/lib/auth.ts

```diff
@@ -38,21 +38,23 @@ export function isPasswordValid(
   password: string,
   breakdown: boolean,
   strict?: boolean
-): { caplow: boolean; num: boolean; min: boolean };
+): { caplow: boolean; num: boolean; min: boolean; admin_min: boolean };
 export function isPasswordValid(password: string, breakdown?: boolean, strict?: boolean) {
   let cap = false, // Has uppercase characters
     low = false, // Has lowercase characters
     num = false, // At least one number
-    min = false; // Eight characters, or fifteen in strict mode.
+    min = false, // Eight characters, or fifteen in strict mode.
+    admin_min = false;
   if (password.length > 7 && (!strict || password.length > 14)) min = true;
+  if (strict && password.length > 14) admin_min = true;
```

Comment:
created a variable admin_min to check valid admin password. 
zod also requires this key("admin_min") to display error if conditions are not met.